src/test/kotlin/org/rust/ide/surroundWith/RustWithBlockSurrounderTest.kt

```diff
@@ -5,6 +5,8 @@ class RustWithBlockSurrounderTest : RustSurrounderTestCaseBase(RustWithBlockSurr
     fun testSimple2() = doTest()
     fun testSimple3() = doTest()
     fun testComments() = doTest()
+    fun testSingleLine() = doTest()
```

Comment:
:+1: yep, this is one of the problems with file based tests: you don't know if you are actually testing the thing